readme.md

```diff
@@ -107,6 +107,20 @@ User can choose between 2 types of installations:
     
     _N.B.: Ports can be customized in advanced options._
 
+## Rebuild static files (for frontend development)
+
+To immediately see your changes (in the `kpi` repository) live you need to run `npm run watch` to rebuild static files on the fly:
+
+```
+$kodo-docker> docker-compose -f docker-compose.frontend.yml -f docker-compose.frontend.override.yml run -p 3000:3000 --rm kpi npm run watch
```

Comment:
Thanks to @jnm work, this could be simplified to:
$kobo-install> ./run.py -cf run -p 3000:3000 --rm kpi npm run 

To use this new option, `two-databases` is merged.

Reply:
probably need some docs related to https://www.flowdock.com/app/kobotoolbox/kobo/threads/oHknJLFhaDrfnSUtNXY2QNoxhCK (maybe applies only to old installs?)